cypress/integration/creatorOnboardingFlows/creatorSettings.spec.js

```diff
@@ -120,4 +89,81 @@ describe('Creator Settings Page', () => {
     cy.findByRole('button', { name: 'Finish' }).click();
     cy.url().should('equal', `${baseUrl}admin/creator_settings/new`);
   });
+
+  context('color contrast ratios', () => {
+    it('should show an error when the constrast ratio of a brand color is too low', () => {
+      const lowContrastColor = '#a6e8a6';
+
+      cy.findByLabelText(/^Brand color/)
+        .clear()
+        .type(lowContrastColor)
+        .trigger('change');
```

Comment:
The change event _should_ be fired, just by executing the `type` method (I can see that it's not though!) 🤔  I wonder if this is a strange thing particular to stimulus

Reply:
I'll take a peek at this @aitchiss.

Reply:
Thanks you both! 🌸 
I tried removing it but the JS doesn't get called without it. 

When @nickytonline gets a chance we can look into it, however, for now I'd like to get this merged once it's approved and we can address it as a separate PR.